resources/styles/components/learning-guide.less

```diff
@@ -30,6 +30,7 @@
     margin: 0 auto;
 
     svg {
+      font-size:12px;
```

Comment:
Can this use the same font size set on the `<html>` (maybe extract to a variable in `variable.less`)?